src/pool/CMakeLists.txt

```diff
@@ -45,10 +45,13 @@ if(UMF_BUILD_LIBUMF_POOL_JEMALLOC)
         NAME jemalloc_pool
         TYPE STATIC
         SRCS pool_jemalloc.c ${POOL_EXTRA_SRCS}
-        LIBS jemalloc ${POOL_EXTRA_LIBS})
+        LIBS ${JEMALLOC_LIBRARIES} ${POOL_EXTRA_LIBS})
     target_include_directories(jemalloc_pool PRIVATE ${JEMALLOC_INCLUDE_DIRS})
     target_compile_definitions(jemalloc_pool
                                PRIVATE ${POOL_COMPILE_DEFINITIONS})
     add_library(${PROJECT_NAME}::jemalloc_pool ALIAS jemalloc_pool)
+    if(NOT WINDOWS)
+        add_dependencies(jemalloc_pool jemalloc)
```

Comment:
Why do we still have `jemalloc_pool`?

Reply:
I am confused, do we have separate library for a `scalable_pool`? I thought no.
For a `disjoint_pool` we have a separate static library because today it is implemented in C++. Once @bratpiorka rewrites it in C it will be part of `libumf.so`.

Reply:
OK, I understand now. I will fix it.

Reply:
Removed.
Done.